src/components/Tutorials/MyTutorials/BaseTutorialsComponent/TutorialCard.js

```diff
@@ -23,7 +23,7 @@ const TutorialCard = ({
           image={
             icon
               ? icon
-              : 'https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png'
+              : "https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
```

Comment:
Better if we can add this to local assets. Not sure what would happen in the future to the image in the URL

Reply:
okay I am moving this to assets folder.